config/config.go

```diff
@@ -737,12 +741,18 @@ func InitServer(ctx context.Context, enabledServices ServerType) error {
 
 	// Set up the server's issuer URL so we can access that data wherever we need to find keys and whatnot
 	// This populates Server.IssuerUrl, and can be safely fetched using server_utils.GetServerIssuerURL()
-	err = parseServerIssuerURL(enabledServices)
+	err = parseServerIssuerURL(currentServers)
 	if err != nil {
 		return err
 	}
 
-	return DiscoverFederation()
+	// Since director is the federation, we need to wait until director's web engine is running
+	// to do DiscoverFederation on director server
```

Comment:
I don't think this logic is right?

If the director is part of a larger federation, then DiscoverFederation may need to be used to discover the registry service.